testing/fixtures/fixtures.go

```diff
@@ -24,20 +24,23 @@ func Directory() string {
 	return filepath.Join(os.TempDir(), "fossa-cli-fixtures")
 }
 
-// Clone executes git clone in target directory and checksout the provided commit, or master if left ""
-func Clone(baseDir string, projects []Project) error {
+// ProjectInitializerFunction defines how a single project should be initialized *after* it has already been cloned
+type ProjectInitializerFunction func(proj Project, projectDir string) error
+
+// InitializeProjects executes git clone in target directory and checksout the provided commit, then runts the initializerFn. This is done asynchronously for each provided project
+func InitializeProjects(baseDir string, projects []Project, initializerFn ProjectInitializerFunction) {
```

Comment:
[nit] I'd call this `Initialize` (`fixtures.InitializeProjects(projects)` isn't much more informative than `fixtures.Initialize(projects)`).